### ISSUE TITLE
WARNING:tensorflow:Learning rate reduction is conditioned on metric `val_loss` which is not available. Available metrics are: loss,acc,lr

### ISSUE THREAD

**Issue by accj1990** [metal3d/keras-video-generators#66]
**System Information**
- your operating system: Colab
- your python version (`python -version`): Python 3.9
- keras video generator version (`pip freeze | grep keras-video-generators`): !pip install git+https://github.com/metal3d/keras-video-generators.git

**Describe the bug**

I get this error message in a code that worked before, has recently stopped working. Can you tell me what might be causing the loss validation not to be calculated ? And how can I adjust it?

7/7 [==============================] - ETA: 0s - loss: 1.4450 - acc: 0.2879WARNING:tensorflow:Early stopping conditioned on metric `val_loss` which is not available. Available metrics are: loss,acc
WARNING:tensorflow:Learning rate reduction is conditioned on metric `val_loss` which is not available. Available metrics are: loss,acc,lr
